packages/shared/components/ProposalAnswer.svelte

```diff
@@ -1,32 +1,39 @@
 <script lang="typescript">
+    import type { Answer } from '@iota/wallet'
     import { createEventDispatcher } from 'svelte'
     import { Text, FontWeight, TooltipIcon } from 'shared/components'
     import { Position } from 'shared/components/enums'
-    import type { Answer } from '@iota/wallet'
-    import { Icon as IconEnum } from '@auxiliary/icon'
+    import { appSettings } from '@core/app/stores'
+    import { Icon } from '@auxiliary/icon'
 
     export let answer: Answer
+    export let answerIndex: number = undefined
+    export let disabled = false
```

Comment:
Can we change this to enabled? I know it's not a real convention, but I think we could start using it more.

Reply:
I think it is easier to standardize as "disabled" because of the HTML native attribute and how the codebase already is
https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/disabled

Reply:
Okay, let's stick to that then. I'm not a big fan of all these negations, but I guess the effort would be too big to change it